fedimint-core/Cargo.toml

```diff
@@ -61,16 +61,16 @@ url = { version = "2.5.2", features = ["serde"] }
 [dev-dependencies]
 test-log = { version = "0.2", features = ["trace"], default-features = false }
 tokio-test = "0.4.4"
-tracing-subscriber = { version = "0.3.18", features = ["env-filter"] }
+tracing-subscriber = { workspace = true }
 
 [target.'cfg(not(target_family = "wasm"))'.dependencies]
-tokio = { version = "1.39.2", features = ["full", "tracing"] }
+tokio = { workspace = true, features = ["full", "tracing"] }
 tokio-rustls = { workspace = true }
 
 [target.'cfg(target_family = "wasm")'.dependencies]
 jsonrpsee-wasm-client = "0.24.2"
 async-lock = "3.4"
-tokio = "1.39.2"
+tokio = { workspace = true, features = ["io-util"] }
```

Comment:
`io-util` needed here? it wasn't there before

Reply:
It is needed, otherwise this line fails:

https://github.com/fedimint/fedimint/blob/cae1f0c6a75262bcd3645b8ee77cc3cbc0c2fdb5/fedimint-core/src/util/mod.rs#L20

I'm not sure why it wasn't needed before